examples/3_ResNet/resnet_infer_python.py

```diff
@@ -96,6 +96,7 @@ def check_results(output: torch.Tensor) -> None:
 
     batch_size_to_run = 1
 
-    result = deploy(saved_model_file, device_to_run, batch_size_to_run)
+    with torch.no_grad():
```

Comment:
I'd suggest using `inference mode` as preferable to `no_grad`

```suggestion
    with torch.inference_mode():
```